share/spack/templates/modules/modulefile.ups_version

```diff
@@ -4,6 +4,7 @@ VERSION = {{spec.version}}
 
 {% if   (spec.architecture|string).find('i686')>0   %}{%set upsbits = '' %}
 {% elif (spec.architecture|string).find('x86_64')>0 %}{%set upsbits = '64bit' %}
+{% else  %}{%set upsbits = '64bit' %}
```

Comment:
Note that the microarchitectures have been receiving quite a lot of love and attention lately upstream. This might need to be revisited when we merge back in from upstream (probably after [0.13](https://github.com/spack/spack/projects/10) is released)